api/src/main/java/org/openmrs/module/fhir/api/strategies/plandefinition/PlanDefinitionStrategyUtil.java

```diff
@@ -0,0 +1,26 @@
+package org.openmrs.module.fhir.api.strategies.plandefinition;
+
+import org.openmrs.Program;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.fhir.api.util.FHIRUtils;
+
+public class PlanDefinitionStrategyUtil {
+
+	private PlanDefinitionStrategyUtil() {
+	}
+
+	public static GenericPlanDefinitionStrategy getPersonStrategy() {
```

Comment:
rename this to getPlanDefinitionStrategy

Reply:
Thanks :)